spec/features/user_views_all_books_page.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'User visits books page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.